### PR TITLE
define swagger schema for custom CoureUser endpoints

### PR DESCRIPTION
**Pull request by docktorrr**
## Purpose
Custom CoreUser endpoints (e.g. coreuser/invite/, coreuser/reset_password/) autogenerete incorrect schema, as default CoreUser serializer is used for it. We need to define custom schema for these endpoints.

## Approach
Use `swagger_auto_schema` decorator for customizing serializers and responses for these endpoints.

### CODE REVIEW

File: workflow/serializers.py [hunk 1]
```diff
@@ -201,8 +201,8 @@ def validate(self, attrs):
 class CoreUserResetPasswordConfirmSerializer(CoreUserResetPasswordCheckSerializer):
     """Serializer for reset password data
     """
-    new_password1 = serializers.CharField(max_length=128, required=True)
-    new_password2 = serializers.CharField(max_length=128, required=True)
```
Comment (ralfzen) [inline workflow/serializers.py:205]: why did you remove the `required=True` here?

**Review by ralfzen — CHANGES_REQUESTED** [review]

File: workflow/serializers.py (hunk 1 above)
Comment (docktorrr) [reply workflow/serializers.py:205]: It's `required=True` by default. We talked about it in Slack.

**Review by ralfzen — APPROVED** [review]

File: workflow/serializers.py (hunk 1 above)
Comment (ralfzen) [reply workflow/serializers.py:205]: ah, right. but anyway, sometimes explicity is better :) but not necessary, of course. so, nicely done 👍
